sklearn/sklearn-template/template/scripts/train.sh

```diff
@@ -51,9 +51,9 @@ PACKAGE_PATH=trainer
 MAIN_TRAINER_MODULE=$PACKAGE_PATH.task
 
 if [ "$RUN_TYPE" = 'hptuning' ]; then
-  CONFIG_FILE=hptuning_config.yaml
+  CONFIG_FILE=configs/hptuning_config.yaml
```

Comment:
Suggest renaming directory to `config`.

Reply:
Modified.

Reply:
I'm not able to see the change. Could you please check?

Reply:
Oh, i have changed the folder name. Forget to change here. Fixed.

Reply:
no worries, thanks